_includes/_boinc_1B.htm

```diff
@@ -96,30 +77,29 @@ <h4 class="text-bold">Manually setting up projects</h4>
 
         <div class="row">
             <div class="col-xs-12">
-                <h4 class="text-bold">BOINC Account Manager Guide</h4>
+                <h4 class="text-bold">BOINC Account Manager Setup</h4>
                 <ul>
                     <li>
                         Create a BAM! account <a href="https://boincstats.com/en/bam/register">here</a>.
                     </li>
                     <li>Wait for the activation mail, and activate your account.</li>
                     <li>Login to BAM! (the login box is at the top-right of <a href="https://boincstats.com">this page</a>).</li>
-                    <li>Navigate to "<a href="https://boincstats.com/en/bam/signup/">Sign-up for projects</a>" in the 'BAM!' menu and click the 'Create account' icon (<img src="/assets/img/BAM.png" alt="BAM logo" />) for every project you want to join.</li>
+                    <li>Go to "<a href="https://boincstats.com/en/bam/signup/">Sign-up for projects</a>" in the 'BAM!' menu and click the 'Create account' icon (<img src="/assets/img/BAM.png" alt="BAM logo" />) for every project you want to join.</li>
```

Comment:
We can drop the BAM logo as the actual image for the create account icon is a paperclip.

Reply:
Oh really? I haven't used BAM so I just assumed that was the right logo